faq.rst

```diff
@@ -399,62 +292,38 @@ How do I compare my local repository to a remote repository?
 To display the list of changes that are in your local repository, but not
 in the remote, use::
 
- hg outgoing <remote repository>
+    $ git log origin/master..
 
 This is the list of changes that will be sent if you call
-``hg push <remote repository>``.  It does **not** include any :ref:`uncommitted
+``git push``.  It does **not** include any :ref:`uncommitted
 changes <hg-status>` in your working copy!
 
 Conversely, for the list of changes that are in the remote repository but
 not in the local, use::
 
- hg incoming <remote repository>
+    $ git log ..origin/master
 
 This is the list of changes that will be retrieved if you call
-``hg pull <remote repository>``.
-
-.. note::
-   In most daily use, you will work against the default remote repository,
-   and therefore simply type ``hg outgoing`` and ``hg incoming``.
-
-   In this case, you can also get a synthetic summary using
-   ``hg summary --remote``.
+``git pull``.
 
 
 How do I update my local repository to be in sync with a remote repository?
 '''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
 
 Run::
 
-   hg pull <remote repository>
+    $ git pull --ff-only <remote> <branch name>
+
+eg::
```

Comment:
"Example" (or something similar) might be a better choice for non-native speakers.